app/models/shelter.rb

```diff
@@ -25,7 +25,7 @@ def average_review
     reviews.average(:rating).to_f
   end
 
-  # def number_applications
-  #   require "pry"; binding.pry
-  # end
+  def number_applications
+    ApplicationPet.joins('JOIN pets on application_pets.pet_id = pets.id').joins('JOIN shelters on shelters.id = pets.shelter_id').where("shelters.id = #{id}").count
```

Comment:
We will see if we can shorten this once we have completed functionality for app.